Does anyone know how my change is related to CI issues:
```
/__w/zephyr/zephyr/subsys/usb/device/class/cdc_acm.c:63:2: error: #warning "USB_CDC_ACM_LOG_LEVEL forced to LOG_LEVEL_NONE" [-Werror=cpp]
   63 | #warning "USB_CDC_ACM_LOG_LEVEL forced to LOG_LEVEL_NONE"
      |  ^~~~~~~
cc1: all warnings being treated as errors
```

--- a/tests/kernel/timer/starve/testcase.yaml
+++ b/tests/kernel/timer/starve/testcase.yaml
@@ -1,5 +1,5 @@
 tests:
   kernel.timer.starve:
     slow: true
-    timeout: 3600
+    timeout: 3700
     tags: kernel timer

